raw_result_diff.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2021, Matthew Olsson <mattco@serenityos.org>
+#
+# SPDX-License-Identifier: MIT
+
+import json
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Tuple
+
+from main import TestResult
+from main import EMOJIS
+
+
+def read_result_files(old_path: Path, new_path: Path) -> Tuple[float, dict, dict]:
```

Comment:
IMO a dataclass would be a better fit here, instead of using a nameless three-element tuple - perhaps using the same names as the variables here: `duration_delta`, `old_results`, `new_results`

Reply:
There are various examples in the main file :)